Stripe/PKPaymentAuthorizationViewController+Stripe_Blocks.m

```diff
@@ -35,22 +38,37 @@ @implementation STPBlockBasedApplePayDelegate
 - (void)paymentAuthorizationViewController:(__unused PKPaymentAuthorizationViewController *)controller
                        didAuthorizePayment:(PKPayment *)payment completion:(STPPaymentAuthorizationStatusCallback)completion {
     self.onPaymentAuthorization(payment);
-    [self.apiClient createTokenWithPayment:payment completion:^(STPToken * _Nullable token, NSError * _Nullable error) {
+
+    void(^tokenOrSourceCompletion)(id<STPSourceProtocol>, NSError *) = ^(id<STPSourceProtocol> result, NSError *error) {
         if (error) {
             self.lastError = error;
             completion(PKPaymentAuthorizationStatusFailure);
             return;
         }
-        self.onTokenCreation(token, ^(NSError *tokenCreationError){
-            if (tokenCreationError) {
-                self.lastError = tokenCreationError;
+        id<STPSourceProtocol> source;
```

Comment:
What is `source` used for? It kind of looks like you meant to pass `source` to the `onSourceCreation()` call, but it's getting `result` right now, so I think this is dead code.

edit: neat, static analyzer agrees with me and failed the build 😄

Reply:
I'm more confident that you meant to pass `source` instead of `result`: down in the definition of `applePaySourceHandler`, the code used to be constructing a `STPPaymentResult` with `token.card`, and now it just passes `source` directly.

Reply:
oh derp, good catch!

Reply:
so i think i understand why this worked (returning `STPToken` instead of the child `STPCard` for an apple pay source). we don't display apple pay sources in the UI, so it didn't matter that `STPToken` doesn't conform to `<STPPaymentMethod>` (the protocol defining how a payment method is displayed). i'll still fix for consistency with how we handle non-apple pay sources.

Reply:
fixed in eb09444, re-r?